.github/workflows/check-strava-api.yml

```diff
@@ -0,0 +1,29 @@
+name: Check Strava API
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  update-model:
+    name: Update Model
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Fetch Latest API
+        uses: stravalib/strava_swagger2pydantic@v1
```

Comment:
Ok wow - you made a new action to do this. this is what i didn't understand. ive never made an action before. i will have to clone that repo to see what the code actually does when it runs.